src/modules/FileSystemUtility.js

```diff
@@ -399,3 +398,53 @@ module.exports.openExternalLink = async (url = "") => {
   if (url === "") return;
   return shell.openExternal(url);
 };
+
+const saveBase64AsImage = async (base64Image) => {
+  const base64Data = base64Image.replace(/^data:image\/\w+;base64,/, "");
+  const buffer = Buffer.from(base64Data, "base64");
+  const fileName = `new_logo.png`; // Adjust the file type as needed
+  const filePath = `./src/assets/${fileName}`;
+  fs.writeFile(filePath, buffer, "base64", (err) => {
+    if (err) {
+      console.error(err);
+      return;
+    }
+  });
+};
+
+module.exports.openImage = async () => {
```

Comment:
This function is unnecessary and should be removed.

Reply:
I need to use openImage function to get the path of image.